### PR TITLE
Ckeditor link functionality

### PR DESCRIPTION
**Pull request by jonasdekeukelaere**
## Type
- Enhancement

## Resolves the following issues
https://github.com/forkcms/forkcms/issues/2524

## Pull request description
- Add "email" type for adding links in ckeditor
- Adds option to make links open in new window when adding links in ckeditor

Adding normal URL:
screen shot 2018-05-16 at 14 03 17

Adding email URL:
screen shot 2018-05-16 at 14 04 28

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,41 +1,29 @@
-version: "2.1"
```
Comment (StijnVrolijk) [inline docker-compose.yml]: This file shouldn't have been changed

Comment (jonasdekeukelaere) [reply docker-compose.yml]: Fixed with commit amend and force push

**Comment by jonasdekeukelaere** [comment]
@StijnVrolijk see description for 2 screenshots

**Comment by ohvitorino** [comment]
This has already been closed and merged for a few days but, it is never to late to say: Nice job @jonasdekeukelaere IMO 👍

**Pull request merged**